addons/property_panel/float_slider/float_slider.gd

```diff
@@ -18,6 +18,8 @@ signal value_changed(value: float)
 @export var step : float
 ## The sensitivity while dragging with the mouse to change the value.
 @export var sensitivity := 1000.0
+## The allow expressions to evaluate to get the value.
+@export var allow_expressions := true
```

Comment:
I thought about this, and I think it makes sense to always allow expressions. It's a nice feature to have, and it isn't dangerous or performance heavy. So IMO it makes sense to just remove the property.